cpp/src/arrow/ipc/reader.cc

```diff
@@ -643,27 +641,6 @@ static Status ReadContiguousPayload(io::InputStream* file,
   return Status::OK();
 }
```

Comment:
Can you remove the `static` qualifier above, since it has become redundant?

Reply:
Thanks @pitrou  for review! removed.